cosmetics-web/spec/models/component_spec.rb

```diff
@@ -373,4 +373,34 @@
       end
     end
   end
+
+  describe "#nano_material_required?" do
+    context "when there is no nano material" do
+      let(:component) { build(:component) }
+
+      it "does not require nano material information" do
+        expect(component).not_to be_nano_material_required
```

Comment:
This might be slightly clearer as:

```suggestion
        expect(component.nano_material_required?).to be false
```

Reply:
I think this is telling us the unary name isn't great. I'd look to improve that, rather than change a unary expectation to a truthy/falsey one 🤷‍♂

Reply:
@baphled `be false` tests by doing `== false`, according to [the docs](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers). The falsey matcher is `be_falsey`.

But you're right that `nano_material_required?` is a bad name... 😩

Reply:
Same principle, tbf. It still reads worse imho.